task_handler_layer/task_buoy/src/buoy_detection.cpp

```diff
@@ -67,12 +67,12 @@ int main(int argc, char *argv[])
 
   ros::init(argc, argv, "buoy_detection");
   ros::NodeHandle n;
-  ros::Publisher pub = n.advertise<std_msgs::Float64MultiArray>("/varun/sensors/front_camera/image_raw", 1000);
+  ros::Publisher pub = n.advertise<std_msgs::Float64MultiArray>("/varun/sensors/front_camera/ip_data", 1000);
```

Comment:
@pkhrag this should publish to `/varun/ip/buoy`. `/varun/sensor/front_camera/ip_data` has senor in its name. The point is to make topic name represent what data is flowing in it.